gui/src/3D/controls/holographicSlate.ts

```diff
@@ -254,20 +297,65 @@ export class HolographicSlate extends ContentDisplay3D {
         return node;
     }
 
+    private _attachContentPlateBehavior() {
+        this._contentDragBehavior.attach(this._contentPlate);
+        this._contentDragBehavior.moveAttached = false;
+        this._contentDragBehavior.useObjectOrientationForDragging = true;
+        this._contentDragBehavior.updateDragPlane = false;
+
+        let origin = new Vector3();

```

Comment:
Actually I think this is better to keep it as local variables, since it is only used as temp storage in the subsequent functions. There are similar patterns in `Behavior`s as well. In that case, it is not putting any strain on the GC since there is no creation of objects (`_attachContentPlateBehavior` is only called once)

Reply:
So my only comment is that most of them can be consts :-)